src/colvarproxy_io.cpp

```diff
@@ -207,14 +207,18 @@ std::istream &colvarproxy_io::input_stream(std::string const &input_name,
   }
 
   if (colvarproxy_io::input_stream_exists(input_name)) {
-    return *(input_streams_[input_name]);
+    std::ifstream *ifs = dynamic_cast<std::ifstream *>(input_streams_[input_name]);
+    if ( ifs && !ifs->is_open()) {
```

Comment:
The case ifs == NULL could mean that the pointer is to a valid istringstream. This method will likely be overridden in Gromacs and whenever other derived istream classes are used, but I'd rather have safeguards against null dereference.

Reply:
Ok with me (the extra check doesn't hurt). However, I would very much to any derived class initializing an entry with a null pointer.

Reply:
So would I, this is just meant to catch null pointers resulting from a failure of dynamic cast.